app/src/main/java/com/cornellappdev/uplift/ui/components/reporting/SubmitButton.kt

```diff
@@ -0,0 +1,45 @@
+package com.cornellappdev.uplift.ui.components.reporting
+
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.util.PRIMARY_BLACK
+import com.cornellappdev.uplift.util.PRIMARY_YELLOW
+import com.cornellappdev.uplift.util.montserratFamily
+
+/**
+ * SubmitButton is a composable that displays a button for the user to submit their report.
+ */
+@Composable
+fun SubmitButton(
+    onSubmit: () -> Unit
+) {
```

Comment:
Kinda an overly looking ahead nitpick, but you could imagine that we'd probably want this button to be unclickable while the report is being sent to backend. If the user could spam it while a report is still being sent, they could spam us with a ton of the same report while not getting feedback that their report is actually being sent.

I'd expose a `enabled: Boolean` variable here that makes the button unclickable and a bit faded (can custom do this with `.alpha(0.5f)`) if true.

Reply:
Not sure if this is what you were thinking, but I added an enabled param of type Boolean and initialized an enabled variable within the ReportIssuesScreen composable that is passed into SubmitButton.